img/a/adt.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 277.53 277.53" width="60" height="60"><path fill="#0061aa" fill-rule="evenodd" d="M81.31 0 0 81.26v115.01l81.31 81.26h114.9l81.32-81.26V81.26L196.21 0H81.31z"/><g fill="#fff"><path d="M142.35 183.69h-30.46V93.83h30.46c25.78 0 45.12 19.08 45.12 44.87s-19.34 44.99-45.12 44.99Zm0-70.27h-10.87v50.56h10.87c14.91 0 25.53-10.24 25.53-25.28s-10.62-25.28-25.53-25.28Z"/><path d="M189.95 185.44v-70.39h-26.49V92.33h76.3v22.72h-26.72v70.39h-23.09z"/></g><path fill="#0061aa" d="M238 94.08v19.22h-26.71v70.39H191.7V113.3h-26.49V94.08h72.8m3.5-3.5h-79.8v26.18h26.49v70.39h26.56v-70.39h26.72V90.58Z"/><path fill="#fff" d="m108.76 185.44-4.65-9.19H66.54l-4.78 9.19h-25.1l48.67-95.25 48.54 95.25Zm-16-31.3-7.44-14.62-7.56 14.62Z"/><path fill="#0061aa" d="M85.32 94.04 131 183.69h-21.18l-4.65-9.19H65.48l-4.78 9.19H39.52l45.8-89.65m-10.41 61.85h20.7l-10.29-20.21-10.41 20.21m10.42-69.55-3.12 6.11-45.8 89.65-2.65 5.09h29l1-1.89 3.8-7.3h35.41l3.68 7.27 1 1.92h29l-2.59-5.09-45.62-89.65-3.11-6.11Zm-4.67 66 4.65-9 4.6 9Z"/></svg>
```

Comment:
Would this be better? I switched the letter outlines to use stroke.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 277.5 277.5" fill="#fff"><path fill="#0061aa" d="M81.3 0 0 81.3v115l81.3 81.2h115l81.2-81.2v-115L196.2 0H81.3z"/><path d="M142.3 183.7H112V93.8h30.4a44 44 0 0 1 45.2 44.9 44 44 0 0 1-45.1 45zm0-70.3h-10.8V164h10.8c15 0 25.6-10.3 25.6-25.3s-10.6-25.3-25.6-25.3z"/><path stroke="#0061aa" stroke-width="3" d="M190 185.4V115h-26.5V92.3h76.3v22.8H213v70.3zm-81.2 0-4.7-9.2H66.5l-4.7 9.2H36.7l48.6-95.2 48.6 95.2Zm-16-31.3-7.5-14.6-7.5 14.6Z"/></svg>
```

Reply:
If "smaller is better", then yes. The question is whether we want to waste our time on optimizing the logos (when they're already small). I think that as long as the various tools do a decent job, there's no apparent need to go further. Once a logo changes, you'd have to do the manual work again anyway.

Since you've already put in many, many hours here, I'll graciously allow it. :wink: